test/validator.test.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint-disable no-undef */
+"use strict";
+import { expect, default as chai } from "chai";
+import sinon from "sinon";
+import chaiAsPromised from "chai-as-promised";
+import { printResults, validateFile } from "../src/validator";
+import {
+  getHappySpec,
+  renderSpecAsFile,
+  getSingleInvalidFile
+} from "./testUtils";
+
+const PROFILE = "mercury";
+
+before(() => {
+  chai.use(chaiAsPromised);
+});
+
+describe("#printResults", () => {
+  let consoleSpy;
+
+  beforeEach(() => {
+    consoleSpy = sinon.spy(console, "log");
+  });
+
+  afterEach(() => {
+    consoleSpy.restore();
+  });
+
+  it("doesn't console log when no results passed", async () => {
+    await printResults(null);
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```

Comment:
Ummmm....what is the error that this was producing and why are we ignoring it?

Reply:
This actually isn't a problem, I resolved it by using this semantic instead of `to.have.been.called`, and then didn't remove this, and then copy pasta. I'll remove.